framework/validators/CompareValidator.php

```diff
@@ -85,28 +85,28 @@ public function init()
         if ($this->message === null) {
             switch ($this->operator) {
                 case '==':
-                    $this->message = Yii::t('yii', '{attribute} must be repeated exactly.');
+                    $this->message = Yii::t('yii', '{attribute} must be equal to "{compareInternal}".');
```

Comment:
Also, when you use compare validator to ensure password is repeated the same, this new message doesn't read very naturally and will almost always be changed.